controllers/comment.js

```diff
@@ -0,0 +1,88 @@
+const express = require('express');
+const {ValidationError} = require('sequelize');
+
+const router = express.Router();
+const responseUtil = require('../helpers/response');
+const { Comment, User } = require('../models');
```

Comment:
kalo ga dipake jangan diimpor yaa,, ini coba dihapus `User`nya karna ga dipake di file ini
```suggestion
const { Comment } = require('../models');
```

Reply:
done